tsl/src/remote/dist_ddl.c

```diff
@@ -841,6 +841,16 @@ dist_ddl_process(ProcessUtilityArgs *args)
 			dist_ddl_process_grant(args);
 			break;
 
+		/*
+		 * "DROP OWNED BY" and "REASSIGN OWNED BY" needs to be shipped to the
+		 * datanodes as is.
+		 */
+		case T_DropOwnedStmt:
+		case T_ReassignOwnedStmt:
+			dist_ddl_state_schedule(DIST_DDL_EXEC_ON_START, args);
```

Comment:
maybe move those to separate functions to be in sync with other part of the code

Reply:
done.